package.json

```diff
@@ -108,6 +108,7 @@
     "react-circular-progressbar": "^2.0.3",
     "react-dom": "^17.0.1",
     "react-draggable": "^4.4.3",
+    "react-feather": "^2.0.9",
```

Comment:
Don't introduce another component for just two icons. You can import svg as we have done it with other icons (see `src/renderer/components/icons/index.ts`

Reply:
@veado I couldn't find moon icon in figma and also in antd. The ticket said to use bepswap moon icon so I checked bepswap code and noticed that they used above component rather than svg.
Can we keep it for now or are you okay to use moon svg file from the google?

Reply:
Pls use a svg file (will be provided via Slack).